components/statuslight/index.css

```diff
@@ -10,68 +10,148 @@ OF ANY KIND, either express or implied. See the License for the specific languag
-    filter: color;
-    replace: --spectrum-statuslight$1-;
-  }
+  --spectrum-statuslight-height: var(--spectrum-component-height-75);
+  --spectrum-statuslight-dot-size: var(--spectrum-status-light-dot-size-small);
+
+  --spectrum-statuslight-font-size: var(--spectrum-font-size-75);
+
+  --spectrum-statuslight-spacing-dot-to-label: var(--spectrum-text-to-visual-75);
+  --spectrum-statuslight-spacing-top-to-dot: var(--spectrum-status-light-top-to-dot-small);
+  --spectrum-statuslight-spacing-top-to-label: var(--spectrum-component-top-to-text-75);
+  --spectrum-statuslight-spacing-bottom-to-label: var(--spectrum-component-bottom-to-text-75);
 }
 
 .spectrum-StatusLight--sizeM {
-  @remapvars {
-    find: /--spectrum-statuslight(.*)-m-/;
-    filter: color;
-    replace: --spectrum-statuslight$1-;
-  }
-}
+  --spectrum-statuslight-height: var(--spectrum-component-height-100);
+  --spectrum-statuslight-dot-size: var(--spectrum-status-light-dot-size-medium);
+
+  --spectrum-statuslight-font-size: var(--spectrum-font-size-100);
+
+  --spectrum-statuslight-spacing-dot-to-label: var(--spectrum-text-to-visual-100);
+  --spectrum-statuslight-spacing-top-to-dot: var(--spectrum-status-light-top-to-dot-medium);
+  --spectrum-statuslight-spacing-top-to-label: var(--spectrum-component-top-to-text-100);
+  --spectrum-statuslight-spacing-bottom-to-label: var(--spectrum-component-bottom-to-text-100);
+ }
 
 .spectrum-StatusLight--sizeL {
-  @remapvars {
-    find: /--spectrum-statuslight(.*)-l-/;
-    filter: color;
-    replace: --spectrum-statuslight$1-;
-  }
+  --spectrum-statuslight-height: var(--spectrum-component-height-200);
+  --spectrum-statuslight-dot-size: var(--spectrum-status-light-dot-size-large);
+
+  --spectrum-statuslight-font-size: var(--spectrum-font-size-200);
+
+  --spectrum-statuslight-spacing-dot-to-label: var(--spectrum-text-to-visual-200);
+  --spectrum-statuslight-spacing-top-to-dot: var(--spectrum-status-light-top-to-dot-large);
+  --spectrum-statuslight-spacing-top-to-label: var(--spectrum-component-top-to-text-200);
+  --spectrum-statuslight-spacing-bottom-to-label: var(--spectrum-component-bottom-to-text-200);
 }
 
 .spectrum-StatusLight--sizeXL {
-  @remapvars {
-    find: /--spectrum-statuslight(.*)-xl-/;
-    filter: color;
-    replace: --spectrum-statuslight$1-;
+  --spectrum-statuslight-height: var(--spectrum-component-height-300);
+  --spectrum-statuslight-dot-size: var(--spectrum-status-light-dot-size-extra-large);
+
+  --spectrum-statuslight-font-size: var(--spectrum-font-size-300);
+
+  --spectrum-statuslight-spacing-dot-to-label: var(--spectrum-text-to-visual-300);
+  --spectrum-statuslight-spacing-top-to-dot: var(--spectrum-status-light-top-to-dot-extra-large);
+  --spectrum-statuslight-spacing-top-to-label: var(--spectrum-component-top-to-text-300);
+  --spectrum-statuslight-spacing-bottom-to-label: var(--spectrum-component-bottom-to-text-300);
+}
+
+@media (forced-colors: active) {
```

Comment:
> Also do we maybe miss a high-contrast mode here for the Status-lights?

I was wondering why there weren't more high contrast mode styles for this component.

@bernhard-adobe is that what you meant,  that it looks like high contrast mode styles are missing?

Reply:
@yosevu @bernhard-adobe I just took a look at the PR deploy in a tool that I have access to to test WHCM in, and Yosevu's work looks the same as what we currently have on the site, so I think the styles here are ok/no changes needed.